webapp-django/crashstats/signature/templates/signature/signature_report.html

```diff
@@ -135,6 +141,34 @@
         </div>
     </section>
 
+    <!-- Graph panel.
+        Displays a graph of crashes/ADU against build date
+    -->
+    <section class="panel" id="graph-panel"
+        data-source-url="{{ url('signature:signature_graph') }}"
+        data-jsonurl="{{ url('signature:signature_report') }}graphdata/"
```

Comment:
There should be an actual name to get that URL. Adding ``graphdata/`` at the end is bad practice.